ark/phenotyping/som_utils_test.py

```diff
@@ -11,115 +12,241 @@
 import ark.utils.test_utils as test_utils
 
 
-def mocked_cluster_pixels(base_dir, chan_list):
-    # assign marker names and create sample marker information
-    sample_pixel_mat = pd.read_csv(os.path.join(base_dir, 'pixel_mat_preprocessed.csv'),
-                                   usecols=chan_list)
+def mocked_train_som(fovs, channels, base_dir,
+                     subset_name='pixel_mat_subsetted.hdf5', weights_name='weights.hdf5'):
+    # define the matrix we'll be training on
+    pixel_mat_sub = pd.DataFrame(columns=channels)
 
-    sample_pixel_mat = sample_pixel_mat
+    for fov in fovs:
```

Comment:
This is really nice. Now the mocked function recapitulates all of the key features of the new version of the real function.